packages/node/src/models/state-channel.ts

```diff
@@ -295,7 +285,7 @@ export class StateChannel {
       userNeuteredExtendedKeys,
       new Map<string, AppInstance>(),
       new Map<string, TwoPartyVirtualEthAsLumpInstance>(),
-      new Map<0, string>(),
+      undefined,
```

Comment:
An empty channel should still have a free balance with the default asset class being ETH (i.e. just invoke `createFreeBalance` here).

Reply:
Before you run setup protocol though you technically do not have a free balance

Reply:
Got it - seems like this is the exception then and it should be noted.